src/extension.ts

```diff
@@ -329,12 +334,43 @@ function launchMetals(
           client.outputChannel.show(true);
           channelOpen = true;
         }
+      },
+      startDebugSession: args => {
+        scalaDebugger.start(args).then(wasStarted => {
+          if (!wasStarted) {
+            window.showErrorMessage("Debug session not started");
+          }
+        });
       }
     };
     Object.entries(clientCommands).forEach(([name, command]) =>
       registerCommand(name, command)
     );
 
+    scalaDebugger
+      .initialize()
+      .forEach(disposable => context.subscriptions.push(disposable));
+    registerCommand(scalaDebugger.startSessionCommand, scalaDebugger.start);
```

Comment:
Can we optionally enable the debugger based on whether the Metals language server supports debugging? We currently do this for the tree view by looking at the server capabilities during `initialize` https://github.com/scalameta/metals-vscode/blob/7a80af4af34c361ca20424ee6b5d950980121d52/src/MetalsFeatures.ts#L20